.buckconfig

```diff
@@ -0,0 +1,6 @@
+
+[android]
+  target = Google Inc.:Google APIs:23
```

Comment:
Is there any reason for using API 23 ? Because it only covers 4.7% of users. (According to android studio)